src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue17390.cs

```diff
@@ -0,0 +1,32 @@
+﻿#if !MACCATALYST
+using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	internal class Issue17390 : _IssuesUITest
+	{
+		public override string Issue => "Shell bottom padding when navigating between tabs";
+
+		public Issue17390(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		[Category(UITestCategories.Shell)]
+		public void ShellBottomPaddingWhenNavigatingBetweenTabs()
```

Comment:
The test is failing on iOS and Windows.
![image](https://github.com/user-attachments/assets/e348a7aa-d717-4f40-b482-954dd964bf45)
Could you review the differences?

Reply:
@jsuarezruiz , We have analyzed the issue and made changes to the image, which should address the failures on both platforms. Please review and let me know if there are any additional concerns.